.travis.yml

```diff
@@ -1,16 +1,28 @@
 language:
   ruby
 
+env:
+  - CPLUS_INCLUDE_PATH=/usr/include/atlas C_INCLUDE_PATH=/usr/include/atlas
+
 rvm:
-  - 1.9.3
-  - 2.0
-  - 2.1
-  - 2.2
+  - '1.9.3'
+  - '2.0'
+  - '2.1'
+  - '2.2'
+
+matrix:
+  fast_finish:
+    true
+
+script: "bundle exec rake test"
+
+install:
+  - gem install bundler
+  - ./.build.sh
+  - bundle install
```

Comment:
But [the Gemfile is empty](https://github.com/SciRuby/statsample/blob/master/Gemfile), so it should be a matter of building and installing the gem... Well, no problem, we can check this later on.